### DISCUSSION TITLE
Strapi on custom context - host:port/my-context

### DISCUSSION THREAD

**Discussion by psmyrdek** [strapi/strapi#7639] [Legacy Topics]
Hi all,

Is it possible to configure *the whole service* in a way it works under `https://localhost:8080/my-context`? 

I'd like to achieve both:
`https://localhost:8080/my-context/admin`
and
`https://localhost:8080/my-context/api`

Having such a configuration I cannot open the admin panel:

```js
module.exports = ({ env }) => ({
  host: 'localhost',
  port: 8080,
  url: '/my-context',
  admin: {
    url: '/admin',
  },
});

```
Thanks for help!

**Comment by derrickmehaffy** [comment]
Currently it's not easily done via just the Strapi config (as we don't have global prefixing) and instead in your case it would be better to use one of our optional software guides and configure one of those to handle it: https://strapi.io/documentation/v3.x/getting-started/deployment.html#optional-software-guides

**Reply by psmyrdek** [reply]
Thanks for quick answer - from the complexity perspective, how hard would it be for me to introduce this change in a form of Pull Reuqest? Is it a well-known request or something not worth investing into due to architectural decisions inside?

**Reply by derrickmehaffy** [reply]
For the moment we are not entertaining global prefixing of routes due to complexity and we just recently reworked the current structure a little less than a month ago. Instead we are offloading that to outside systems like proxy applications.

**Reply by psmyrdek** [reply]
Got it, thanks.
